addons/hr_holidays/tests/test_holidays_flow.py

```diff
@@ -118,31 +103,6 @@ def _check_holidays_status(holiday_status, ml, lt, rl, vrl):
         # Case2: limited type of leave request
```

Comment:
We could split this into two flows, using the comments, you'd have one testing in the case of an hr.leave.type which would not be limited and another with limited hr.leave.type